try{
    throw new Error("yeah....sorry");
}
catch(error){
    console.log(error)
}